### DISCUSSION TITLE
Do we have plan to support multi teacher or multi-head students?

### DISCUSSION THREAD

**Discussion by LeighSu** [yoshitomo-matsubara/torchdistill#371] [Ideas]
Hello everyone, this project is very excellect tool. 
So do we have plan to support multi teacher or multi-head students?

**Comment by yoshitomo-matsubara** [comment]
Hi @LeighSu 

It is technically possible as discussed [here](https://github.com/yoshitomo-matsubara/torchdistill/discussions/244#discussioncomment-3480211).
